pallets/pallet-dyn-filter/src/mock.rs

```diff
@@ -0,0 +1,198 @@
+pub(crate) type Hash = sp_core::H256;
+pub(crate) type Signature = MultiSignature;
+pub(crate) type AccountPublic = <Signature as Verify>::Signer;
+pub(crate) type AccountId = <AccountPublic as IdentifyAccount>::AccountId;
+pub(crate) type Index = u64;
+pub(crate) type BlockNumber = u64;
+
+frame_support::construct_runtime!(
+	pub enum Test where
+		Block = Block,
+		NodeBlock = Block,
+		UncheckedExtrinsic = UncheckedExtrinsic,
+	{
+		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
+		DynFilter: crate::{Pallet, Storage, Call, Event<T>},
+	}
+);
+
+parameter_types! {
+	pub const SS58Prefix: u8 = 38;
+	pub const BlockHashCount: BlockNumber = 2400;
+}
+
+impl frame_system::Config for Test {
+	type BaseCallFilter = frame_support::traits::Everything;
+	type BlockWeights = ();
+	type BlockLength = ();
+	type DbWeight = ();
+	type Origin = Origin;
+	type Call = Call;
+	type Index = Index;
+	type BlockNumber = BlockNumber;
+	type Hash = Hash;
+	type Hashing = BlakeTwo256;
+	type AccountId = AccountId;
+	type Lookup = IdentityLookup<Self::AccountId>;
+	type Header = Header;
+	type Event = Event;
+	type BlockHashCount = BlockHashCount;
+	type Version = ();
+	type PalletInfo = PalletInfo;
+	type AccountData = ();
+	type OnNewAccount = ();
+	type OnKilledAccount = ();
+	type SystemWeightInfo = ();
+	type SS58Prefix = SS58Prefix;
+	type OnSetCode = ();
+	type MaxConsumers = frame_support::traits::ConstU32<16>;
+}
+
+impl crate::Config for Test {
+	type Event = Event;
+	type WeightInfo = ();
+
+	type EnsureOrigin = EnsureRoot<AccountId>;
+	type FeatureCall = FeatureCalls;
+	type GovernanceCall = GovCalls;
+	type StakingCall = StakingCalls;
+	type TransferCall = TransferCalls;
+	type XcmCall = XcmCalls;
+	type SystemCall = SystemCalls;
+}
+
+pub(crate) const GOVERNANCE: &[u8] = b"gov";
+pub(crate) const STAKING: &[u8] = b"stk";
+pub(crate) const TRANSFER: &[u8] = b"trf";
+pub(crate) const FEATURE: &[u8] = b"fet";
+pub(crate) const XCM: &[u8] = b"xcm";
+pub(crate) const SYSTEM: &[u8] = b"system";
+
+lazy_static! {
+	pub static ref CALL_GOV: Call = Call::System(frame_system::Call::remark {
+		remark: GOVERNANCE.to_vec()
+	});
+	pub static ref CALL_STAKING: Call = Call::System(frame_system::Call::remark {
+		remark: STAKING.to_vec()
+	});
+	pub static ref CALL_TRANSFER: Call = Call::System(frame_system::Call::remark {
+		remark: TRANSFER.to_vec()
+	});
+	pub static ref CALL_FEATURE: Call = Call::System(frame_system::Call::remark {
+		remark: FEATURE.to_vec()
+	});
+	pub static ref CALL_XCM: Call = Call::System(frame_system::Call::remark { remark: XCM.to_vec() });
+	pub static ref CALL_SYSTEM: Call = Call::System(frame_system::Call::remark {
+		remark: SYSTEM.to_vec()
+	});
+}
+
+pub struct GovCalls;
+impl Contains<Call> for GovCalls {
+	fn contains(t: &Call) -> bool {
+		if let Call::System(frame_system::Call::remark { remark }) = t {
+			return &remark[..] == GOVERNANCE;
+		}
+		return false;
```

Comment:
S: Get rid of `return` statements 
```suggestion
		if let Call::System(frame_system::Call::remark { remark }) = t {
			&remark[..] == GOVERNANCE
		} else {
			false
		}
```

Reply:
[7d2ddc0](https://github.com/KILTprotocol/kilt-node/pull/397/commits/7d2ddc05ea3d21a15a0f766444934b2c8c37ac97)